snapshot/git/gitdb/db_test.go

```diff
@@ -337,12 +337,26 @@ func TestBundlestore(t *testing.T) {
 
 	consumerDB := MakeDBFromRepo(consumerDataRepo, fixture.tmp, streamCfg, nil, bundleCfg, AutoUploadBundlestore)
 
+	upstreamMaster, err := fixture.upstream.RunSha("rev-parse", "master")
```

Comment:
It's getting onerous to read through longer tests like these. Can you sprinkle in some comments? And in particular, how does this test the bundlestore.go change?